tendrl/integrations/gluster/check_cluster_status.py

```diff
@@ -0,0 +1,23 @@
+import etcd
+
+
+def run():
+    try:
+        clusters = NS._int.client.read("/clusters")
+    except etcd.EtcdKeyNotFound:
+        return
+
+    for cluster in clusters.leaves:
+        int_id = cluster.key.split('/')[-1]
+        try:
+            NS._int.client.write(
```

Comment:
do this only for is_managed == "yes" clusters

Reply:
Ah ok. will change the logic and update